### PR TITLE
chore(CPlusPlus): display longest name

### PR DESCRIPTION
**Pull request by samughodake**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

File: algorithms/CPlusPlus/Strings/longest-name.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  string names[] = {
+    "hi",
+    "hello",
+    "helloall",
+    "helloeveryone"
+  };
+  //calculating size of the array
+  int n = sizeof(names) / sizeof(names[0]);
+  cout << "The longest string in the array is: " << longest(names, n);
+}
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Strings/longest-name.cpp:38]: add return 0 to tell that your program runs successfully

**Review by UG-SEP — CHANGES_REQUESTED** [review]
Please make the changes shown below

File: algorithms/CPlusPlus/Strings/longest-name.cpp (hunk 1 above)
Comment (Arsenic-ATG) [reply algorithms/CPlusPlus/Strings/longest-name.cpp:38]: The body of the `main ()` does not need to contain the return statement, if control reaches the end of main without encountering a return statement, the effect is that of executing `return 0;` 

Although it is recommended to explicitly add it for readability reasons but is not mandatory.

Comment (samughodake) [reply algorithms/CPlusPlus/Strings/longest-name.cpp:39]: Added return 0 to main.
Thanks

**Review by UG-SEP — APPROVED** [review]
Great work @samughodake

**Pull request merged**